common/model-views.cpp

```diff
@@ -3710,6 +3710,10 @@ namespace rs2
 
     bool device_model::check_for_bundled_fw_update(const rs2::context &ctx, std::shared_ptr<notifications_model> not_model , bool reset_delay )
     {
+        // The viewer application supply a bundled FW version (binary data bundled inside the application binary file) when the user run CMake with a
```

Comment:
```
// LibRS can have a "bundled" FW binary downloaded during CMake. That's the version
// "available" to us, but it may not be there (e.g., no internet connection to download
// it). Lacking an available version, we try to let the user choose a "recommended"
// version for download. The recommended version is defined by the device (and comes
// from a #define).
```